### PR TITLE
chore(web): fly to layers

### PR DESCRIPTION
**Pull request by mkumbobeaty**
# Overview

## What I've done

- Add flyto UI
- Implement zoom to layer on click flyto button

### CODE REVIEW

File: web/src/beta/features/Editor/tabs/map/LeftPanel/Layers/index.tsx [hunk 1]
```diff
@@ -84,11 +99,23 @@ const LayerContainer = styled.div`
...
+`;
+
 const AddLayerIcon = styled.div`
   padding: 2px;
   margin-bottom: 2px;
   align-self: flex-end;
   cursor: pointer;
 `;
-
+const ZoomLayer = styled.div``;
```
Comment (KaWaite) [inline web/src/beta/features/Editor/tabs/map/LeftPanel/Layers/index.tsx:114]: Don't need

File: web/src/beta/lib/core/Map/hooks.ts [hunk 2]
```diff
@@ -112,6 +116,13 @@ export default function ({
     );
   }, [onLayerSelect, selectedLayer]);
 
+  useEffect(() => {
+    if (zoomedLayerId) {
+      engineRef.current?.lookAtLayer(zoomedLayerId);
+      onZoomToLayer?.(undefined);
+    }
+  }, [zoomedLayerId, onZoomToLayer]);
```
Comment (KaWaite) [inline web/src/beta/lib/core/Map/hooks.ts:124]: This is alreadying happening in core/Visualizer/hooks.ts. Any reason it needs to happen in Map too?

Comment (mkumbobeaty) [reply web/src/beta/lib/core/Map/hooks.ts:124]: removed, sorry I forget to clean it

Comment (KaWaite) [reply web/src/beta/lib/core/Map/hooks.ts]: Please remove zoomLayerId and onZoomToLayer from the prop type too 🙏🏻

**Review by KaWaite — APPROVED** [review]

**Review by keiya01 — APPROVED** [review]

**Pull request merged**